src/Thujohn/Twitter/Traits/AccountActivityTrait.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Thujohn\Twitter\Traits;
+
+use BadMethodCallException;
+
+trait AccountActivityTrait
+{
+	/**
+	 * Creates HMAC SHA-256 hash from incomming crc_token and consumer secret.
+	 * This base64 encoded hash needs to be returned by the application when Twitter calls the webhook.
+	 *
+	 * @param  mixed $crcToken
+	 *
+	 * @return void
+	 */
+	public function CrcHash($crcToken)
```

Comment:
lowercase first char please :)
```suggestion
	public function crcHash($crcToken)
```